cfpb/cfgov-refresh
# Re-adds gulp-bless to split stylesheets for IE

`gulp-bless` was removed due to a [CSS-parsing bug](https://github.com/cfpb/cfgov-refresh/pull/5137#issuecomment-517743389). This PR fixes the bug by patching its upstream dependency and adding it to package.json as a [selective resolution](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/).

## Additions

- `gulp-bless`
- `resolutions` section `package.json`

## Testing

1. Pull down branch
1. `./frontend.sh`
1. View http://0.0.0.0:8000/ in legacy versions of IE and the carousel should load.

## Screenshots

<img width="1367" alt="Screen Shot 2020-02-19 at 1 15 19 PM" src="https://user-images.githubusercontent.com/1060248/74862686-94449880-531a-11ea-9dce-6aeb84a8bfc7.png">
